python/python/bystro/prs/tests/test_preprocess_for_prs.py

```diff
@@ -70,7 +79,7 @@ def mock_dosage_df():
 def mock_dosage_df_clean():
     return pd.DataFrame(
         {
-            "locus": ["chr1:566875:C:T", "chr1:728951:A:G", "chr1:917492:C:T"],
+            "locus": ["chr8:132782505:C:T", "chr3:183978846:G:A", "chr2:4000400:T:C"],
```

Comment:
These loci should all be included in the SNPID and both ID_effect_as_alt and ID_effect_as_ref in mock_processed_scores_df()